raritan/decorators.py

```diff
@@ -144,6 +144,7 @@ def input_data(*args, **kwargs):
     A callable which is the original function with decoration.
     """
     analyze = kwargs.get('analyze', True)
+    optional_flag = kwargs.get('optional', False)
```

Comment:
Does this mean that you can set an entire @input_data as optional? Something like:

```
@input_data(optional=True)
def load_foo():
...
```

Reply:
yeah i think that'd work too because there are functions in our main flow that are like get_dictionaries()